test/Fail/Issue5681.err

```diff
@@ -1,6 +1,6 @@
 Failed to solve the following constraints:
   Sort _2 of foo admits data/record definitions. (blocked on _2)
-  Is usable at default modality: foo (blocked on _2)
+  Is usable at strictly positive modality: foo (blocked on _2)
```

Comment:
Here I do not see why `foo` is to required to be usable at the _strictly positive_ modality.